sources/install/package_base.sh

```diff
@@ -173,6 +173,14 @@ function install_ultimate_vimrc() {
     sh ~/.vim_runtime/install_awesome_vimrc.sh
 }
 
+function install_neovim() {
+    curl -LO https://github.com/neovim/neovim/releases/latest/download/nvim.appimage
+    chmod u+x nvim.appimage
+    ./nvim.appimage --appimage-extract
+    cp -r squashfs-root/usr/* /usr/
```

Comment:
If this is an extraction of binaries, please extract to the `/opt/tools/bin/` directory